docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -325,18 +331,24 @@ Release
 
 #. Build RPM packages and upload them to ``archive.clusterhq.com``
 
+   .. note:: Skip this step for a documentation release.
+
    .. code-block:: console
 
       admin/upload-rpms "${VERSION}"
 
 #. Build and upload the tutorial :ref:`Vagrant box <build-vagrant-box>`.
 
+   .. note:: Skip this step for a documentation release.
```

Comment:
This one means that the "Access" at the top to "Vagrant Cloud" is not necessary. Note also (make an issue?) that that link is also old now that Atlas is a thing.